tools/SourceKit/tools/sourcekitd/bin/InProc/CMakeLists.txt

```diff
@@ -1,10 +1,15 @@
 set(EXPORTED_SYMBOL_FILE "${SOURCEKITD_SOURCE_DIR}/bin/sourcekitd.exports")
+set(SOURCEKITD_INPROC_DEPEND SourceKitSwiftLang sourcekitdAPI)
+
+if(SOURCEKIT_NEED_EXPLICIT_LIBDISPATCH)
+  set(SOURCEKITD_INPROC_DEPEND ${SOURCEKITD_INPROC_DEPEND} dispatch)
+endif()
```

Comment:
I'm no CMake expert, so I'm curious: which is better here, using a list or using this sort of expansion? Are they the same thing?